frontend/src/scenes/trends/ActionsLineGraph.js

```diff
@@ -16,7 +15,7 @@ export function ActionsLineGraph({ dashboardItemId = null, filters: filtersParam
     }, [toParams(otherFilters)])
 
     return results ? (
-        results.reduce((total, item) => total + item.count, 0) > 0 ? (
+        filters.session || results.reduce((total, item) => total + item.count, 0) > 0 ? (
```

Comment:
Referring to comment above. I opted to use a bunch of spot if/elses like this to reduce possible collision with existing logic.